CareerJSM/docker-rails
# Publish images to github packages when pushing to master or creating tags

## Description
Changed the Github workflow to store images in the Github Packages image registry and added pushing tags as a trigger for the workflow.

## Related Links
https://app.asana.com/0/1200379515898093/1200385916593831

## Testing
* Create tags following our tag naming scheme and observe the build runs and publishes the appropriate image to the Github Packages registry
* Observe we build and publish for all of the architectures in the matrix when pushing to master

## Docs
[Authorizations](https://github.com/CareerJSM/careerjsm-code/blob/develop/docs/reference/authorization.md) | [Change Yarn Version](https://github.com/CareerJSM/careerjsm-code/blob/develop/docs/reference/change-yarn-version.md) | [Packages](https://github.com/CareerJSM/careerjsm-code/blob/develop/docs/onboarding/packages.md) | [Debugging](https://github.com/CareerJSM/careerjsm-code/blob/develop/docs/procedures/debugging.md) | [Product](https://github.com/CareerJSM/careerjsm-code/blob/develop/docs/onboarding/product.md) | [System Setup](https://github.com/CareerJSM/careerjsm-code/blob/develop/docs/onboarding/quick-guide-system-setup.md)
